GitEbachS/Cooking-in-Season
# css finalized, sorting on ingredients fixed, data complete

CSS has a proper flow throughout the app. The ingredients are properly sorted in ascending order. The sign in page has a new picture.


## Description
The ingredients are sorted in ascending order so that when a user adds 'Step 3' after 'Step 5' has been created, all the steps will show in the correct number sequence.

## Motivation and Context
The app needed a proper flow of colors, layouts, and fonts. Also, the ingredients needed to use the sort() method so that they would appear in numeric order, even after the user updates certain steps.

## How Can This Be Tested?
CSS is finalized. You can click on any link in the navbar to see each page. The color schemes, fonts, buttons, and page layouts all flow together. 
To view the steps for the ingredients, click on the view recipe button on the recipe card.

## Screenshots (if appropriate):

## Types of changes

- [x ] Bug fix (non-breaking change which fixes an issue)
- [x ] New feature (non-breaking change which adds functionality)
